### PR TITLE
Accept invites

### PR DESCRIPTION
**Pull request by chloeprezelski**
This PR allows you to accept and reject invites to a pod. 

When you get invited to a pod, the invite will render in the pods page when you go there. If you choose to hide the modal, you can bring it back up with the "show invites" button to take action regarding the invite.

### CODE REVIEW

File: api/src/routes/invites.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { isConstructorDeclaration } from "typescript";
+import Pod from "../../models/Pod";
+import PodInvites from "../../models/PodInvites";
+import User from "../../models/User";
+import auth, { AuthRequest } from "../authMiddleware";
+
+let invitesRouter = express.Router();
+
+invitesRouter.get(
+  "/currUsersInvites",
```
Comment (zusoomro) [inline api/src/routes/invites.ts]: I think we should be consistent between this route and GET /events - either by changing GET /events to GET /events/currUserEvents or by changing this route to just GET /invites

File: api/src/routes/invites.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      res.status(500).send("Server Error");
+    }
+  }
+);
+
+invitesRouter.post(
+  "/accept",
+  [auth],
+  async (req: express.Request, res: express.Response) => {
+    console.log("hit endpoint /invites/accept");
```
Comment (zusoomro) [inline api/src/routes/invites.ts:54]: !!

File: api/src/routes/invites.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      res.status(500).send("Server Error");
+    }
+  }
+);
+
+invitesRouter.post("/reject", async (req, res) => {
+  const { id } = req.body;
+  try {
+    const numDeleted = await PodInvites.query().deleteById(id);
+    res.send({ msg: "success" });
```
Comment (zusoomro) [inline api/src/routes/invites.ts]: This is another thing we should try to be consistent with across the board. Should we use msg or message? I've used message a bit throughout the rest of the api

File: mobile/yarn.lock [hunk 4]
```diff
@@ -8520,6 +8526,8 @@ react-native-status-bar-height@^2.5.0:
   resolved "https://registry.yarnpkg.com/react-native-status-bar-height/-/react-native-status-bar-height-2.5.0.tgz#bc0fb85230603850aab9667ee8111a62954de90c"
   integrity sha512-sYBCPYA/NapBSHkdm/IVL4ID3LLlIuLqINi2FBDyMkc2BU9pfSGOtkz9yfxoK39mYJuTrlTOQ7mManARUsYDSA==
 
+<<<<<<< HEAD
```
Comment (zusoomro) [inline mobile/yarn.lock]: Looks like you have some merge markers in your yarn.lock. Maybe try using the instructions in the README to resolve the issues?

**Review by zusoomro — APPROVED** [review]
Looks good! I think we're going to have to rewrite a little bit of this because of the pod data refactoring but it looks solid

File: api/src/routes/invites.ts (hunk 3 above)
Comment (chloeprezelski) [reply api/src/routes/invites.ts]: Changed to "message" :)

File: api/src/routes/invites.ts (hunk 1 above)
Comment (chloeprezelski) [reply api/src/routes/invites.ts]: changed to invites/ to match /events/

File: api/src/routes/invites.ts (hunk 2 above)
Comment (chloeprezelski) [reply api/src/routes/invites.ts:54]: ! ?

**Pull request merged**